components/PasswordDialog.qml

```diff
@@ -283,7 +286,14 @@ Item {
                 MoneroComponents.StandardButton {
                     id: okButton
                     small: true
-                    text: qsTr("Ok") + translationManager.emptyString
+                    text: {
```

Comment:
`okButton` text should be set outside.. Would prefer to introduce new optional `open` function parameter.
Same for `rightIcon` property.

Reply:
Submitted https://github.com/monero-project/monero-gui/pull/3111 (co-authored with you).

Reply:
Ok, thanks. I'll update this PR accordingly.